src/server.ts

```diff
@@ -13,5 +16,61 @@ export default function createServer(opts?: Fastify.ServerOptions) {
     return { twitterHandle: request.query.handle }
   })
 
+
+  // Fastify Swagger Test URI demonstration i.e. sample Path. 
```

Comment:
Move this `.post` block to the configureSwaggerPlugin function

Reply:
Done.